### PR TITLE
Fix inaccuracy in HTMLTableRowElement doc

### PR DESCRIPTION
**Pull request by jackdeguest**
…nguage in HTMLTableElement

#### Metadata

- [ ] Adds a new document
- [X] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/htmltablerowelement/index.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ _Inherits properties from its parent, {{domxref("HTMLElement")}}._
 _Inherits methods from its parent, {{domxref("HTMLElement")}}_.
 
 - {{domxref("HTMLTableRowElement.deleteCell()")}}
```
Comment (sideshowbarker) [inline files/en-us/web/api/htmltablerowelement/index.md:32]:
```suggestion
- {{domxref("HTMLTableRowElement.deleteCell()", "deleteCell(index)")}}
```

File: files/en-us/web/api/htmltablerowelement/index.md [hunk 2]
```diff
@@ -30,9 +30,9 @@ _Inherits properties from its parent, {{domxref("HTMLElement")}}._
...
-  - : Removes the cell at the given position in the row. If it is equal to `-1`, it deletes the last cell of the row. If the given position is greater (or equal as it starts at zero) than the amount of cells in the row, or is smaller than `-1`, it raises a {{domxref("DOMException")}} with the `IndexSizeError` value.
+  - : Removes the cell corresponding to the `index` given in parameter. If the `index` value is `-1` the last cell of the row is removed; if it is smaller than `-1` or greater than the amount of cells in the collection, a {{DOMxRef("DOMException")}} with the value `IndexSizeError` is raised.
 - {{domxref("HTMLTableRowElement.insertCell()")}}
```
Comment (sideshowbarker) [inline files/en-us/web/api/htmltablerowelement/index.md:34]:
```suggestion
- {{domxref("HTMLTableRowElement.insertCell()", "insertCell(index)")}}
```

File: files/en-us/web/api/htmltablerowelement/index.md [hunk 3]
```diff
@@ -29,10 +29,10 @@ _Inherits properties from its parent, {{domxref("HTMLElement")}}._
...
 
-- {{domxref("HTMLTableRowElement.deleteCell()")}}
-  - : Removes the cell at the given position in the row. If it is equal to `-1`, it deletes the last cell of the row. If the given position is greater (or equal as it starts at zero) than the amount of cells in the row, or is smaller than `-1`, it raises a {{domxref("DOMException")}} with the `IndexSizeError` value.
-- {{domxref("HTMLTableRowElement.insertCell()")}}
-  - : Inserts a new cell just before the given position in the row. If the given position is not given or is `-1`, it appends the cell to the row. If the given position is greater (or equal as it starts at zero) than the amount of cells in the row, or is smaller than `-1`, it raises a {{domxref("DOMException")}} with the `IndexSizeError` value. Returns a reference to a {{domxref("HTMLTableCellElement")}}.
+- {{domxref("HTMLTableRowElement.deleteCell()", "deleteCell(index)")}}
+  - : Removes the cell corresponding to the `index` given in parameter. If the `index` value is `-1` the last cell of the row is removed; if it is smaller than `-1` or greater than the amount of cells in the collection, a {{DOMxRef("DOMException")}} with the value `IndexSizeError` is raised.
```
Comment (sideshowbarker) [inline files/en-us/web/api/htmltablerowelement/index.md:33]:
```suggestion
  - : Removes the cell corresponding to `index`. If `index` is `-1`, the last cell of the row is removed; if `index` is smaller than `-1` or greater than the amount of cells in the collection, a {{DOMxRef("DOMException")}} with the value `IndexSizeError` is raised.
```

File: files/en-us/web/api/htmltablerowelement/index.md [hunk 4]
```diff
@@ -29,10 +29,10 @@ _Inherits properties from its parent, {{domxref("HTMLElement")}}._
...
+- {{domxref("HTMLTableRowElement.insertCell()", "insertCell(index)")}}
+  - : Returns an {{DOMxRef("HTMLTableCellElement")}} representing a new cell of the row. It inserts it in the cells collection immediately before the given `index` position in the row. If the `index` is `-1`, the new cell is appended to the collection. If the `index` is smaller than `-1` or greater than the number of cells in the collection, a {{DOMxRef("DOMException")}} with the value `IndexSizeError` is raised.
```
Comment (sideshowbarker) [inline files/en-us/web/api/htmltablerowelement/index.md:35]:
```suggestion
  - : Returns an {{DOMxRef("HTMLTableCellElement")}} representing a new cell of the row. The cell is inserted in the collection of cells immediately before the given `index` position in the row. If `index` is `-1`, the new cell is appended to the collection. If `index` is less than `-1` or greater than the number of cells in the collection, a {{DOMxRef("DOMException")}} with the value `IndexSizeError` is raised.
```

File: files/en-us/web/api/htmltablerowelement/index.md (hunk 3 above)
Comment (jackdeguest) [reply files/en-us/web/api/htmltablerowelement/index.md:33]: I wholeheartedly agree with your modification, but you would need to apply those throughout the other pages of the documentation for HTMLTableElement and others as well.

File: files/en-us/web/api/htmltablerowelement/index.md (hunk 4 above)
Comment (jackdeguest) [reply files/en-us/web/api/htmltablerowelement/index.md]: Of course, I agree and like I mentioned previously, those modifications should be applied to other parts of the table documentations for consistency.

File: files/en-us/web/api/htmltablerowelement/index.md [hunk 5]
```diff
@@ -29,10 +29,10 @@ _Inherits properties from its parent, {{domxref("HTMLElement")}}._
...
 
-- {{domxref("HTMLTableRowElement.deleteCell()")}}
-  - : Removes the cell at the given position in the row. If it is equal to `-1`, it deletes the last cell of the row. If the given position is greater (or equal as it starts at zero) than the amount of cells in the row, or is smaller than `-1`, it raises a {{domxref("DOMException")}} with the `IndexSizeError` value.
-- {{domxref("HTMLTableRowElement.insertCell()")}}
-  - : Inserts a new cell just before the given position in the row. If the given position is not given or is `-1`, it appends the cell to the row. If the given position is greater (or equal as it starts at zero) than the amount of cells in the row, or is smaller than `-1`, it raises a {{domxref("DOMException")}} with the `IndexSizeError` value. Returns a reference to a {{domxref("HTMLTableCellElement")}}.
+- {{domxref("HTMLTableRowElement.deleteCell()", "deleteCell(index)")}}
+  - : Removes the cell corresponding to `index`. If `index` is `-1`, the last cell of the row is removed; if `index` is less than `-1` or greater than the amount of cells in the collection, a {{DOMxRef("DOMException")}} with the value `IndexSizeError` is raised.
```
Comment (sideshowbarker) [inline files/en-us/web/api/htmltablerowelement/index.md:33]:
```suggestion
  - : Removes the cell corresponding to `index`. If `index` is `-1`, the last cell of the row is removed. If `index` is less than `-1` or greater than the amount of cells in the collection, a {{DOMxRef("DOMException")}} with the value `IndexSizeError` is raised.
```

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**